core/drivers/stm32_ipcc.c

```diff
@@ -217,7 +220,6 @@ stm32_ipcc_pm(enum pm_op op, unsigned int pm_hint,
 		panic();
 
 	if (!PM_HINT_IS_STATE(pm_hint, CONTEXT) || !is_tdcid)
-		return TEE_SUCCESS;
 
```

Comment:
Restore the return instruction (see commit "drivers: stm32_ipcc: do not make RIF configuration mandatory").